OurUmbraco.Site/Views/Partials/Projects/ListProjects.cshtml

```diff
@@ -185,7 +185,7 @@
                                 defaultScreenshot = childContent.GetPropertyValue("defaultScreenshotPath", false, "/css/img/package2.png");
                             }
                         }
-                        <img src="@Utils.GetScreenshotPath(defaultScreenshot)?bgcolor=fff&width=50&height=50&format=png" />
+                        <img src="@Utils.GetScreenshotPath(defaultScreenshot)?bgcolor=fff&width=100&height=100&format=png" />
                     </div>
```

Comment:
Hm, I have no problem implementing srcset, but for us Surface users, it doesn't look like Edge supports it either. I'm not sure what the stats are for Our on what browser, though. My whole team uses Surfaces (I'm on a Surface Book), but I think we all use Chrome.

It would be good to have access to that data to see who it might effect!